community_supplied/Protocols/Bgp/bgp-kpi.playbook

```diff
@@ -8,10 +8,15 @@
  *    and notifies when anomalies are found.
  * 3) Rule "check-forwarding-table-netconf" Check if peer ip is present in forwarding table
  *    aand provides vector data to check-peer rule.
+ * 4) Rule "check-bgp-config-netconf" monitors the BGP neighbor configurations.
+ * 5) Rule "check-bgp-summary-netconf.rule" monitors the BGP neighbor summary and notifies
+      when anomalies are found.
+ * 6) Rule "check-ecmp-active-paths-netconf.rule" monitors the health of ECMP active path
+ *    count in control plane and notifies when anomalies are found.
  */
 healthbot {
     playbook bgp-kpi-community {
```

Comment:
make playbook name and file name as bgp-additional-kpis

Reply:
renamed and updated it.